app/modules/ads/setup.js

```diff
@@ -142,7 +142,7 @@ export const adsSetup = {
     context.set('options.slotRepeater', isGeoEnabled('wgAdDriverRepeatMobileIncontentCountries'));
 
     context.set('services.browsi.enabled', isGeoEnabled('wgAdDriverBrowsiCountries'));
-    context.set('services.confiant.enabled', isGeoEnabled('wgAdDriverConfiantCountries'));
+    context.set('services.confiant.enabled', isGeoEnabled('wgAdDriverConfiantMobileCountries'));
```

Comment:
Wouldn't it be more solid if we keep check for both variables? Then remove the old one 24h after the release on two new ones.

Reply:
If you worried about the cache then it's fine. But you right, there might be a case if someone edit old variable. I'll add old variable here too, thanks! :)